### ISSUE TITLE
Certificate Error When Sending Telegram Notifications

### ISSUE THREAD

**Issue by TheSpad** [containrrr/shoutrrr#54]
```
root@docker:~$ docker run containrrr/shoutrrr:latest send telegram://<redacted>@telegram?channels=-1000000000000 "Test Message"
URL: telegram://<redacted>@telegram?channels=-1000000000000
Message: Test Message
error invoking send: Post https://api.telegram.org/bot<redacted>/sendMessage: x509: certificate signed by unknown authority
```
Providing an invalid token gives the same result:
```
root@docker:~$ docker run containrrr/shoutrrr:latest send telegram://1234567:A@telegram?channels=-1000000000000 "Test Message"
URL: telegram://1234567:A@telegram?channels=-1000000000000
Message: Test Message
error invoking send: Post https://api.telegram.org/bot1234567:A/sendMessage: x509: certificate signed by unknown authority
```
The host machine has no such trust issues:
```
root@docker:~$ curl https://api.telegram.org/bot1234567:A/sendMessage
{"ok":false,"error_code":401,"description":"Unauthorized"}
```

**Comment by piksel** [comment]
Yeah, that is probably an issue with the shoutrrr docker image.

**Comment by piksel** [comment]
This should be fixed as per `v0.2`!
